app/_includes/nav-v2.html

```diff
@@ -28,6 +28,14 @@
     <div class="separator"></div>
     <div class="navbar-items" role="navigation" aria-label="Main menu">
       <ul class="navbar-items" role="menubar">
+        <li role="menuitem" class="main-menu-item hiring-badge other">
```

Comment:
What is "other"? I'm not seeing it pick up any styling.

Reply:
```suggestion
        <li role="menuitem" class="main-menu-item hiring-badge"> 
``` 

This was me trying to trigger a rebuild - forgot to remove it. Good catch!